Index: ../server/main.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/../server/main.py b/../server/main.py
--- a/../server/main.py	
+++ b/../server/main.py	
@@ -6,6 +6,7 @@
 from uuid import UUID, uuid4 as getId
 from pydantic import BaseModel
 from typing import List
+from fastapi.middleware.cors import CORSMiddleware
 
 NOTES = {
     "C": [16.35, 32.7, 65.41, 130.81, 261.63, 523.25, 1046.5, 2093, 4186, ],
@@ -52,6 +53,18 @@
 # ser = serial.Serial('/dev/ttyACM0', 9600)
 app = FastAPI()
 
+origins = [
+    "http://localhost:3000",
+]
+
+app.add_middleware(
+    CORSMiddleware,
+    allow_origins=origins,
+    allow_credentials=True,
+    allow_methods=["*"],
+    allow_headers=["*"],
+)
+
 def pointFromModel(model: PointModel) -> SoundPoint:
     soundFrequency = NOTES[model.note][model.octave]
     sound = Sound(model.duration, soundFrequency)
@@ -61,6 +74,7 @@
 
 @app.post("/playComposition/")
 async def submit_composition(model: CompositionModel):
+    print(model)
     points = list(map(pointFromModel, model.points))
     composition = Composition(points)
 
